lib/cloudkeeper/aws/proto_helper.rb

```diff
@@ -3,78 +3,52 @@ module Aws
     # Module refining basic GRPC structs with additional methods
     #   used for conversion from one format to another
     module ProtoHelper
+      APPLIANCE_VALUES = %w[identifier title
+                            description mpuri
+                            group ram
+                            core version
+                            architecture operating_system
+                            vo expiration_date
+                            image_list_identifier base_mpuri
+                            appid digest].freeze
+
       refine CloudkeeperGrpc::Appliance do
         # This method converts Appliance class into array of tags, that
         # can be used as AWS tags.
         def to_tags
-          [{ key: 'cloudkeeper_appliance_identifier', value: identifier },
-           { key: 'cloudkeeper_appliance_title', value: title },
-           { key: 'cloudkeeper_appliance_description', value: description },
-           { key: 'cloudkeeper_appliance_mpuri', value: mpuri },
-           { key: 'cloudkeeper_appliance_group', value: group },
-           { key: 'cloudkeeper_appliance_ram', value: ram },
-           { key: 'cloudkeeper_appliance_core', value: core },
-           { key: 'cloudkeeper_appliance_version', value: version },
-           { key: 'cloudkeeper_appliance_architecture', value: architecture },
-           { key: 'cloudkeeper_appliance_operating_system', value: operating_system },
-           { key: 'cloudkeeper_appliance_vo', value: vo },
-           { key: 'cloudkeeper_appliance_expiration_date', value: expiration_date },
-           { key: 'cloudkeeper_appliance_image_list_identifier', value: image_list_identifier },
-           { key: 'cloudkeeper_appliance_base_mpuri', value: base_mpuri },
-           { key: 'cloudkeeper_appliance_appid', value: appid },
-           { key: 'cloudkeeper_appliance_digest', value: digest }] + image.to_tags
+          tags = APPLIANCE_VALUES.map { |v| { key: "cloudkeeper_appliance_#{v}", value: send(v) } }
```

Comment:
Remove code duplications and add `cloudkeeper_applaince` and `cloudkeeper_image` as constants.